x/staging/posts/types/msgs.go

```diff
@@ -127,21 +124,20 @@ func (msg MsgEditPost) ValidateBasic() error {
 	}
 
 	if len(strings.TrimSpace(msg.Message)) == 0 && len(msg.Attachments) == 0 && msg.Poll == nil {
-		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest,
-			"post message, attachments or poll are required and cannot be all blank or empty")
+		return ErrInvalidEmptyFields
```

Comment:
I think this should be `ErrInvalidPost`

```suggestion
		return sdkerrors.Wrap(types.ErrInvalidPost,
			"post message, attachments or poll are required and cannot be all blank or empty")
```